src/web/mjs/connectors/Baozimh.mjs

```diff
@@ -0,0 +1,47 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class Baozimh extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'baozimh';
+        super.label = '包子漫書 (baozimh)';
+        this.tags = [ 'manga', 'webtoon', 'chinese' ];
+        this.url = 'https://www.baozimh.com/';
+    }
+	
+    async _getMangaFromURI(uri) {
+        const request = new Request(uri, this.requestOptions);
+		const data = await this.fetchDOM(request, '.comics-detail__title'); //title object
+        const id = uri.pathname;
+		let title = data[0].textContent.trim()
+        return new Manga(this, id, title);
+    }
+
+    async _getMangas() {
+        let msg = 'This website does not provide a manga list, please copy and paste the URL containing the images directly from your browser into HakuNeko.';
+        throw new Error(msg);
+    }
+
+    async _getChapters(manga) {
+        let request = new Request(new URL(manga.id, this.url), this.requestOptions);
+		//where can it find the chapters
+		const dom = await this.fetchDOM(request);
```

Comment:
MAy use query directly as second parameter `this.fetchDOM(request, '.l-box #chapter-items > div > a, .l-box #chapters_other_list > div > a')`